scripts/requirements.txt

```diff
@@ -1,4 +1,5 @@
 Flask
 mailmanclient
 gunicorn
+validate-email-address
```

Comment:
pyDNS wird auch benötigt:

For check the mx records or check if the email exists you must'
Exception: For check the mx records or check if the email exists you must have installed pyDNS python package